crates/dojo-lang/src/plugin.rs

```diff
@@ -238,6 +241,9 @@ impl MacroPlugin for BuiltinDojoPlugin {
         item_ast: ast::ModuleItem,
         _metadata: &MacroPluginMetadata<'_>,
     ) -> PluginResult {
+        let do_expand: bool =
+            std::env::var(DOJO_PLUGIN_EXPAND_VAR_ENV).map_or(false, |v| v == "true" || v == "1");
+
```

Comment:
What's the purpose behind this env var? Is there a reason to print out the expanded code ?

Reply:
Debugging mostly.

While working on `dojo-lang` to introduce the array, I found that having the compiler rewritten node expanded was very useful.